mcux/drivers/imx/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ zephyr_library_compile_definitions_ifdef(
 
 zephyr_sources_ifdef(CONFIG_HAS_MCUX_CACHE    fsl_cache.c)
 zephyr_sources_ifdef(CONFIG_DISPLAY_MCUX_ELCDIF	fsl_elcdif.c)
+zephyr_sources_ifdef(CONFIG_VIDEO_MCUX_ELCDIF  fsl_elcdif.c)
```

Comment:
why do we need a different Kconfig symbol, cant you use CONFIG_DISPLAY_MCUX_ELCDIF?

Reply:
We don't need different symbols, actually after #21009 merge [1], we should not need the CONFIG_DISPLAY_MCUX_ELCDIF anymore here. The point is that display_mcux_elcdif driver will be moved to video_mcux_elcdif driver.

[1] https://github.com/zephyrproject-rtos/zephyr/pull/21009